src/config_utils.cpp

```diff
@@ -335,6 +336,21 @@ void ConfigUtils::initUnsetPropertiesWithDefaults(Config& config)
     INIT_UNSET_PROPERTY(config.addonOptions.dualDirectionalOptions, dpadMode, static_cast<DpadMode>(DUAL_DIRECTIONAL_STICK_MODE));
     INIT_UNSET_PROPERTY(config.addonOptions.dualDirectionalOptions, combineMode, DUAL_DIRECTIONAL_COMBINE_MODE);
 
+		// addonOptions.tiltOptions
```

Comment:
These lines are indented with two tabs, the rest of the file is indented with 4 spaces. Please adjust your added lines accordingly.